tests/EvmAcceptanceTests/start_ganache.sh

```diff
@@ -0,0 +1,2 @@
+clear
```

Comment:
can you create an issue for porting this script to a [hardhat task](https://hardhat.org/hardhat-runner/docs/advanced/create-task) and assign it to me?

Reply:
we can use hardhat-ganache plugin as well.

Reply:
https://zilliqa-jira.atlassian.net/browse/ZIL-4770